aries-site/src/pages/components/footer.js

```diff
@@ -18,20 +23,155 @@ const Footer = () => (
-          figma="https://www.figma.com/file/9aoCaf5lqzdQv1q4NFi0GN/hpe-design-system-library-footer"
+          code="https://raw.githubusercontent.com/hpe-design/design-system/master/aries-site/src/examples/components/footer/FooterExample.js"
+          figma="https://www.figma.com/file/9aoCaf5lqzdQv1q4NFi0GN/HPE-Footer-Component?node-id=84%3A26"
           width="100%"
         >
           <FooterExample />
         </Example>
       </Subsection>
     </ContentSection>
+    <ContentSection>
+      <Subsection name="Guidance">
+        <SubsectionText>
+          Footer is used at the bottom of an application. It provides buttons
+          that link to company information and other important links to
+          secondary pages that may not be part of the{' '}
+          <Anchor label="Header" href={nameToPath('Header')} /> navigation
+          structure.
+        </SubsectionText>
+      </Subsection>
+      <Subsection name="Usage" level={3} gap="small">
+        <SubsectionText>
+          A footer is a consistent element to be used at the bottom of all of
+          your applications or webpages.
+        </SubsectionText>
+        <SubsectionText>
+          The most common footer is an{' '}
+          <Anchor label="App Footer" href="#app-footer" /> which is contains
+          legal and other important resources that pertain to the entirety of an
+          application.
+        </SubsectionText>
+        <SubsectionText>
+          The other type of footer is a{' '}
+          <Anchor label="Page Footer" href="#page-footer" /> which contains
+          additional actions or resources specific to a single page of an
+          application.
+        </SubsectionText>
+      </Subsection>
+      <Subsection name="App Footer" level={3}>
+        <SubsectionText>
+          A simple, clean and straightforward app footer displays useful
+          information for the user that couldn’t be displayed on the header.
+          This Footer is typically present on every screen of an application and
+          its contents are universal to the entire application.
+        </SubsectionText>
+        <Text weight="bold">
+          Always provide these essential elements in your app footer:
+        </Text>
+        <BulletedList
+          items={[
+            'Copyright information',
+            'Terms of Use',
+            'Privacy',
+            'Security',
+          ]}
+        />
+        <SubsectionText>
+          Check out this <Anchor href="#" label="simple Footer example" /> to
+          see how these essential elements are included within a Footer.
+        </SubsectionText>
+      </Subsection>
+      <Subsection name="Page Footer" level={3}>
+        <SubsectionText>
+          A page-level Header provides additional resources or functions for a
+          single page within an application.
+        </SubsectionText>
+        <Text weight="bold">A page-level Header should:</Text>
```

Comment:
again confused on header ?

Reply:
yea, my bad, I kind of referenced on header. I can change that on my text

Reply:
Nevermind, I don't see it on the text I wrote??

Reply:
Fixed!